bears/js/JSHintBear.py

```diff
@@ -142,8 +142,9 @@ def generate_config(filename, file,
         :param allow_grouping_operator:
             This option allows the use of the grouping operator when it is
             not strictly required.
-        :param prohibit_variable_statements:
-            This option forbids the use of VariableStatements.
+        :param allow_var_statement:
+            Allows the use of the ``var`` statement while declaring variable.
+            Use ``let`` or ``const`` otherwise.
```

Comment:
what do you mean otherwise? its a little unclear. otherwise as in if its False and not true?